tests/Aspire.Hosting.Azure.Tests/AzureContainerAppsTests.cs

```diff
@@ -307,10 +307,10 @@ public async Task ProjectWithManyReferenceTypes()
           "path": "api.module.bicep",
           "params": {
             "api_containerport": "{api.containerPort}",
-            "mydb_secretoutputs_connectionstring": "{mydb.secretOutputs.connectionString}",
+            "mydb_secretoutputs": "{mydb.secretOutputs}",
```

Comment:
So is the assumption here if we have a placeholder which ends in ...secretOutputs that we are referring to the keyvault name?

Reply:
Not assumption, that's the behavior we spec for azure bicep resources. 

```
{resource.secretOutputs} = secret output keyvault name
{resource.secretOutputs.{name}} - secret value
```

We have some existing conventions that are similar from the GA release (but aren't as clean) e.g. if a resource has a paramter called "keyVaultName" and an empty string value, we will pass the secret output keyvault name for that resource. In the future, that would just be:

```JSON
{
    "params":  {
        "keyVaultName": "{resource.secretOutputs}"
     }
}
```

That's the more explicit way to define without assuming anything about the names of parameters.